docs/administration-guide/13-embedding.md

```diff
@@ -65,3 +65,7 @@ Dashboards are a fixed aspect ratio, so if you'd like to ensure they're automati
 
 ### Reference applications
 To see concrete examples of how to embed Metabase in applications under a number of common frameworks, check out our [reference implementations](https://github.com/metabase/embedding-reference-apps) on Github.
+
+
+## Premium embedding
+If you'd like to embed Metabase dashboards or charts in your application without the "Powered by Metabase" attribution, you can purchase premium embedding from the Metabase store. [Find out more here](http://store.metabase.com/).
```

Comment:
Not sure where this link should go, probably a blog post or a propaganda page. @kdoh do you think landing on the product page makes the most sense or should we have a different spot?

Reply:
Product page probably makes the most sense.